### DISCUSSION TITLE
Week 7

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#161] [General]
<h3>Debugging I can expect to use in the near future... </h3>
The youtube video related to this week's debugging exercises because a human error of flipping the wrong switch, caused the Apollo computer to show error signs. In FreeCodeCamp's debugging exercises, multiple activities were based on re-reading code and identifying spelling and bracket/quotation errors. Spelling errors are something that I commonly fall into and without fail am both embarrassed and relieved after yelling at my computer that it was such a simple fix. I plan on using console.log() often in my future coding because it's so useful. 
<h3>Final Project</h3>
I think as the course has progressed I've found myself drawn to the project idea of creating a simple, online menu for a burger restaurant. This page will be similar to the Cafe Menu we coded weeks prior and I intend to use the experience to base my own. When my partner asks how the course has been going I respond that I love HTML and CSS, but don't enjoy Java Script. After this week's lessons in ES6 and debugging, I definitely felt more comfortable, but not nearly to the level that I feel with HTML/Java.

**Comment by Skye404** [comment]
JavaScript is definitely more challenging than HTML and CSS. I'm finding that I'm thrown into learning so much at once with JavaScript, so I don't retain much or feel like I'm learning, but then the next week things get more clear because some knowledge is sticking, after all. 

You could do a lot with a restaurant menu to really spice it up and make it look more professional than the simple Cafe Menu activity we did. It sounds like a fun project!
